python/tchannel/statsd.py

```diff
@@ -45,6 +47,17 @@ def before_send_request(self, request):
 
         self._statsd.count(key, 1)
 
+    def before_send_request_per_attempt(self, request, retry_count):
+        statsd_name = "tchannel.outbound.calls.retries"
+        retry_count += 1
+        key = common_prefix(statsd_name, request) + '.' + str(retry_count)
```

Comment:
We shouldn't be generating complete metric streams for individual attempt numbers.